api/api/src/test/scala/uk/ac/wellcome/platform/api/works/WorksTestDeleted.scala

```diff
@@ -0,0 +1,48 @@
+package uk.ac.wellcome.platform.api.works
+
+import uk.ac.wellcome.models.Implicits._
+import uk.ac.wellcome.models.work.internal.WorkState.Indexed
+import uk.ac.wellcome.models.work.internal._
+
+class WorksTestDeleted extends ApiWorksTestBase {
+  val deletedWork: Work.Deleted[Indexed] = indexedWork().deleted()
+
+  it("returns an HTTP 410 Gone if looking up a deleted work") {
+    withWorksApi {
+      case (worksIndex, routes) =>
+        insertIntoElasticsearch(worksIndex, deletedWork)
+        val path = s"/$apiPrefix/works/${deletedWork.state.canonicalId}"
+        assertJsonResponse(routes, path) {
+          Status.Gone -> deleted(apiPrefix)
+        }
+    }
+  }
+
+  it("excludes deleted works from list results") {
+    withWorksApi {
+      case (worksIndex, routes) =>
+        val works = indexedWorks(count = 2).sortBy {
+          _.state.canonicalId
+        }
+
+        val worksToIndex = Seq[Work[Indexed]](deletedWork) ++ works
+        insertIntoElasticsearch(worksIndex, worksToIndex: _*)
```

Comment:
Maybe simpler:

```suggestion
        insertIntoElasticsearch(worksIndex, deletedWork, works: _*)
```

Reply:
Unfortunately the compiler doesn't like that cause it can't figure out the common type between `deletedWork` and `works`. I think this is why it was done like this in `WorkTestInvisible`

Reply:
I figured there'd be a sensible reason, thanks for explaining.